scheduler/dev-config.edn

```diff
@@ -1,6 +1,17 @@
 {:port 12321
  ;; We'll set the user to vagrant, since that's the default for many Vagrant-based Mesos setups
  :authorization {:one-user "vagrant"}
+
+ ;; What function should be used to perform user authorization?
+ ;; See the docstring in cook.authorization for details.
+ :authorization-fn cook.authorization/configfile-admins-auth
```

Comment:
Yea, I think that is a reasonable restriction with the benefit being that we have clear modularity.